scripts/make.py

```diff
@@ -534,6 +562,17 @@ def compiler_str(bin_name: str) -> str:
         if cfg.is_cross_compiling:
             self._copy_runtime_to_native(lib_spec)
 
+        # For newlib, the --with-target-subdir configure option is passed by default
+        # Due to that target libraries are normally built in a subdirectory whose
+        # name is the argument to --target (ie, here it is getting built in the path
+        # "INSTALL_DIR/lib/clang-runtimes/{lib_spec_name}/{target}".
+        # Copying these built libraries and headers into the correct path
+        # "INSTALL_DIR/lib/clang-runtimes/{lib_spec_name}"
+
+        self._copy_newlib_headers_and_libs(join(cfg.target_llvm_rt_dir, lib_spec.name, lib_spec.target,                                                   'lib'), join(cfg.target_llvm_rt_dir, lib_spec.name, 'lib'))
```

Comment:
the 'lib' here is all the way out on the same line. I'm guessing that should probably be on the next line or on the same line without the spaces in between